ffi_derive/src/lib.rs

```diff
@@ -266,47 +266,62 @@
 //! ```
 //!
 
+#![deny(unused_extern_crates, missing_docs)]
 #![warn(
+    box_pointers,
     clippy::all,
     clippy::correctness,
     clippy::nursery,
     clippy::pedantic,
     future_incompatible,
     missing_copy_implementations,
     nonstandard_style,
+    rust_2018_idioms,
     trivial_casts,
     trivial_numeric_casts,
-    unreachable_pub,
     unused_qualifications,
     unused_results,
     variant_size_differences
 )]
-#![forbid(missing_docs, unused_extern_crates, unused_imports)]
+#![allow(box_pointers)]
```

Comment:
Just remove line 271?

Reply:
🤦  But it's more fun to turn them on then back off!